### PR TITLE
updated the entries HTML endpoint

### PR DESCRIPTION
**Pull request by collin**
@mparent61 

* filters entries by published
* takes an array of entry ids rather than a timestamp

### CODE REVIEW

**Comment by collin** [comment]
Will merge this into the main PR, just wanted an isolated review.

File: bulbs/liveblog/views.py [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
 
 
-class LiveblogNewEntriesView(TemplateView):
+class LiveblogNewEntriesView(BaseContentDetailView):
+    model = LiveBlogModel
     template_name = 'liveblog/new_entries.html'
+    redirect_correct_path = False
 
-    def get_context_data(self, slug, pk):
+    def get_context_data(self, object):
```
Comment (mparent61) [inline bulbs/liveblog/views.py]: should you call parent (super) class method first? (Do you want any of parent's context values?)

File: bulbs/liveblog/views.py [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
-        when = dateparse.parse_datetime(self.request.GET['new_as_of'])
-        if not when:
-            raise ValueError('param new_as_of must be an ISO formatted timestamp')
-        context['entries'] = liveblog.entries.filter(published__gte=when)
+        if 'entry_ids' not in self.request.GET:
+            raise ValueError('param "entry_ids" MUST be specified')
+        parsed_entry_ids = [x.strip() for x in self.request.GET['entry_ids'].split(',')]
+        context['entries'] = self.object.entries.filter(
+            pk__in=parsed_entry_ids,
+            published__lte=timezone.now())
```
Comment (mparent61) [inline bulbs/liveblog/views.py]: Does this also filter when "published=None" ? (my None comparison logic needs refresher)

File: bulbs/liveblog/views.py (hunk 1 above)
Comment (collin) [inline bulbs/liveblog/views.py]: No, don't need any of the other data, special coverage stuff etc.

**Comment by mparent61** [comment]
LGTM

File: bulbs/liveblog/views.py (hunk 2 above)
Comment (collin) [inline bulbs/liveblog/views.py]: @mparent61 added a test that confirms `published=None` are filtered out.

**Pull request merged**